edx_username_changer/tasks.py

```diff
@@ -23,7 +23,7 @@
 User = get_user_model()
 
 
-@task()
+@shared_task()
```

Comment:
FYI: we can't use retry here because the transactions in this task are not atomic and if any specific part breaks we don't want the task to be run all again and cause more complications.